apps/server/src/modules/oauth/uc/oauth.uc.ts

```diff
@@ -125,7 +125,8 @@ export class OauthUc {
 			const schoolToMigrate: SchoolDO | null = await this.schoolMigrationService.schoolToMigrate(
```

Comment:
There should be a test where you can add the systemId in the toHaveBeenCalled

Reply:
toHaveBeenCalled test done